### ISSUE TITLE
gr.Examples() not working fine with gr.Image(tool='sketch')

### ISSUE THREAD

**Issue by PouriaRouzrokh** [gradio-app/gradio#2348]
### Describe the bug

Hi and thank you for developing this very interesting tool!

I am working on an interface to showcase an inpainting algorithm, where the user should use the sketchpad to draw a mask on an image which they have already uploaded to the interface. For this, I am using gr.Image(tool="sketch) in the interface.

Everything works fine, but when I try to provide some images as examples for the users, the gr.Example() does not look to do it job appropriately. The example image thumbnail is loaded into the interface but when I click on it to show the loaded example in the Image component, only a gray screen is shown instead of the image itself. This works normally when I remove the tool="sketch" argument from the gr.Image() command.

In other words, the following lines of codes work fine:

```Python
img = gr.Image(image_mode='L', type='pil', label='Input image').style(height=300)
gr.Examples(['test.png'], inputs=[img])
```

But the following lines, result in an inappropriate image shown (look the uploaded image below):

```Python
img = gr.Image(tool='sketch', image_mode='L', type='pil', label='Input image').style(height=300)
 gr.Examples(['test.png'], inputs=[img])
```

Is there any workaround for this?

Thank you!

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Reproduction

` img = gr.Image(tool='sketch', image_mode='L', type='pil', label='Input image').style(height=300)
  gr.Examples(['test.png'], inputs=[img])`

### Screenshot

This image shows the state of the Image component, after I click on the example thumbnail. Naturally, it should load the image, but it only shows a gray screen:

![image](https://user-images.githubusercontent.com/70486383/192568976-4066f8f0-edba-4232-978c-ece1422cf199.png)

### Logs

```shell
No errors received.
```

### System Info

```shell
3.3.1, Linux server
```

### Severity

blocking all usage of gradio

**Comment by fffiloni** [comment]
Have you tried the new sketch tools released in 3.4 ? —› [Take a look to the new features](https://github.com/gradio-app/gradio/releases) ;)

What you can try to make it works:
- update your app to 3.4
- replace your input by: `img = gr.Image(source="upload", tool='sketch', image_mode='L', type='pil', label='Input image').style(height=300)`
- replace `type="pil"`by `type="numpy"`
- To load Exmaples, if your using the uploaded image with binary mask, you should provide the original image AND the corresponding mask as two different arguments, for example : `inputs=[image, mask]`

Tell me if that helps :)

**Comment by freddyaboulton** [comment]
Hi @PouriaRouzrokh !

I think this has been fixed. I am able to upload a background image into a sketch component via examples in gradio version 3.17.0!

![image_example_with_mask](https://user-images.githubusercontent.com/41651716/216156152-bd3ae3d0-1627-4ef1-9c44-b97b7f26c42b.gif)

Code:

```python
import gradio as gr

with gr.Blocks() as demo:
    with gr.Row():
        with gr.Column():
            img = gr.Image(tool='sketch', image_mode="L",
                                      type='pil', label='Input image', interactive=True).style(height=300)
            submit = gr.Button(value="Submit")
        with gr.Column():
            img_2 = gr.Image()
            mask = gr.Image()
    submit.click(lambda x: (x['image'], x['mask']), inputs=[img], outputs=[img_2, mask])
    gr.Examples(['avatar.jpg'], inputs=[img])

demo.launch()
```

Will close. Thank you!
